quadra/tasks/segmentation.py

```diff
@@ -341,13 +341,15 @@ def test(self) -> None:
             for batch in dataloader:
                 images, masks, labels = batch
                 images = images.to(self.device)
+                if "16" in str(self.deployment_model.model_dtype):
```

Comment:
This can be problematic for the future considering `bfloat16` or `float16-true` for now it is okay.

Reply:
Instead of .half we could call images.to(self.deployment_model.model_dtype), but isn't the check still valid for bfloat and float16-true?

Reply:
We can have a TODO comment to check for the future updates. I haven't tried it for other combinations but at least if an error occurs, we will know where to look at first.